### PR TITLE
Modularization of mod11

### LINKED ISSUE #3: Modulo 11 in its own file/module
Could be nice to have no external deps, so don't use an existing mod11. But we now have mod11 in two files, extract it to one file would be nice.

### PR DESCRIPTION
**Pull request by tanettrimas**
Fixes #3 

Applied with the comments given in #7

- kontonr.js now calls mod11_generate and mod11_validate instead of re-implementing the same logic. 
- Added validation of kontonummer
- mod11 and applyWeights are now squashed
- mod11_generate only generate and do not do any checking
 
Note: Ref #7, getDefaultWeights() cannot remove the reversed because then the validation of kontonummer will fail.

### CODE REVIEW

File: src/mod11.js [hunk 1]
```diff
@@ -8,12 +8,20 @@ function gen(n) {
 /**
  * NOTE: caller of this function must handle the case where this function returns 10.
  *       typically 10 should be '-' when used as control digit.
```
Comment (ergor) [inline src/mod11.js]: This note is made obsolete by this PR, it should be removed:

```suggestion
```

File: src/mod11.js [hunk 2]
```diff
@@ -35,9 +43,9 @@ function getDefaultWeights(n) {
     const weightOf = idx => ((idx + 6) % 6) + 2;
 
     return Array
-            .from({length: n}, (val, idx) => idx) // => [0, 1, .., n-1]
-            .reverse()                            // => [n-1, .., 1, 0]
-            .map(val => weightOf(val));           // => [w(n-1), .., 3, 2]
+        .from({length: n}, (_, idx) => idx) // => [0, 1, .., n-1]
+        .reverse()                            // => [n-1, .., 1, 0]
+        .map(val => weightOf(val));           // => [w(n-1), .., 3, 2]
```
Comment (ergor) [inline src/mod11.js]: Comments should be aligned for readability
```suggestion
        .from({length: n}, (_, idx) => idx) // => [0, 1, .., n-1]
        .reverse()                          // => [n-1, .., 1, 0]
        .map(val => weightOf(val));         // => [w(n-1), .., 3, 2]
```

File: src/orgnr.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ function gen() {
 	if (contr === 11) contr = 0;
 
 	if (contr === 10)
-       return null;
+    return null;
```
Comment (ergor) [inline src/orgnr.js:20]: Looks like spaces snuck in instead of tab, should be tab here:
```suggestion
	return null;
```

**Review by ergor — CHANGES_REQUESTED** [review]
Looks very good! There is only some whitespace stuff and a dangling comment that should to be addressed, and it's good to go

File: src/orgnr.js (hunk 3 above)
Comment (tanettrimas) [reply src/orgnr.js:20]: This is resolved in #13

File: src/mod11.js (hunk 2 above)
Comment (tanettrimas) [reply src/mod11.js]: Fixed by c240816

File: src/mod11.js (hunk 1 above)
Comment (tanettrimas) [reply src/mod11.js]: Fixed by c240816

**Review by tanettrimas — COMMENTED** [review]
Should be fixed by c240816

**Review by ergor — APPROVED** [review]
A nice upgrade to the mod11 lib

**Pull request merged**